io.github.cxong.cdogs-sdl.json

```diff
@@ -0,0 +1,74 @@
+{
+  "app-id": "io.github.cxong.cdogs-sdl",
+  "runtime":"org.freedesktop.Platform",
+  "runtime-version":"20.08",
+  "sdk":"org.freedesktop.Sdk",
+  "command": "cdogs-sdl",
+  "finish-args": [
+      "--device=dri",
+      "--share=ipc",
+      "--socket=pulseaudio",
+      "--socket=wayland",
+      "--socket=fallback-x11",
+      "--share=network"
+  ],
+  "modules": [
+    {
+      "name": "protobuf",
+      "cleanup": [
+        "protoc",
+        "/bin",
+        "/doc",
+        "/lib/plugins"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://github.com/protocolbuffers/protobuf/releases/download/v3.6.1/protobuf-cpp-3.6.1.tar.gz",
+          "sha256": "b3732e471a9bb7950f090fd0457ebd2536a9ba0891b7f3785919c654fe2a2529"
+        }
+      ]
+    },
+
+
+        {
+        "name": "python3-protobuf",
+        "buildsystem": "simple",
+        "build-commands": [
+            "python3 setup.py install --prefix=/app --root=/"
+        ],
+        "sources": [
+            {
+                "type": "archive",
+                "url": "https://files.pythonhosted.org/packages/3d/64/a3b379cb9c7827ad33c67dcda4c4ad117bdef1b7d68b22a05c963cf4727d/protobuf-3.17.3.tar.gz",
+                "sha256": "72804ea5eaa9c22a090d2803813e280fb273b62d5ae497aaf3553d141c4fdd7b"
+            }
+        ]
+    },
+      {
+          "name": "cdogs-sdl",
+          "buildsystem": "simple",
+          "build-commands": [
```

Comment:
You could split between a `shell` source to run before build and a `post-install`, and use `cmake-ninja`